src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ConfigDumpReferenceCommandTest.php

```diff
@@ -115,6 +115,17 @@ public function testDumpAtPathXml(bool $debug)
         $this->assertStringContainsString('[ERROR] The "path" option is only available for the "yaml" format.', $tester->getDisplay());
     }
 
+    #[TestWith(['yaml'])]
+    #[TestWith(['xml'])]
+    public function testDumpFrameworkBundle(string $format): void
+    {
+        $tester = $this->createCommandTester(true);
+        $ret = $tester->execute(['name' => 'framework', '--format' => $format]);
```

Comment:
This integration test increases the coverage of the `(Yaml|Xml)ReferenceDumper` classes with the configuration of the FrameworkBundle.